client/ayon_core/plugins/publish/extract_review.py

```diff
@@ -978,6 +1203,14 @@ def input_output_paths(self, new_repre, output_def, temp_data):
             # Make sure to have full path to one input file
             full_input_path_single_file = full_input_path
 
+        filled_files = temp_data.get("filled_files", {})
```

Comment:
Isn't the key always filled?
```suggestion
        filled_files = temp_data["filled_files"]
```